common/tools/js-mcp-server/README.md

```diff
@@ -0,0 +1,66 @@
+# Azure SDK MCP Server
+
+A Model Context Protocol (MCP) server implementation for the Azure SDK repository. This server is designed to facilitate the development and testing of tools that communicate using the MCP.
```

Comment:
Why not just call it mcp-server instead of adding js to it?

Reply:
I was trying to avoid conflating this with the office [Azure MCP](https://github.com/Azure/azure-mcp) - any better names you can think of? Happy to call it whatever y'all think makes sense. It's technically under the `@azure-tools` scope so even if we do end up publishing it at some point there would be no conflict (just maybe some confusion)

Reply:
Well, that's the Azure MCP server versus Azure SDK for JS MCP, hopefully that's wordy enough to distinguish.

Reply:
Sounds good. Renamed to `@azure-tools/mcp-server`